src/resources/appContext.json

```diff
@@ -73,7 +73,8 @@
 					"maxResolution": 0.29858214169740677,
 					"minResolution": 156543.03390625,
 					"rotation": 0.0,
-					"projection": "EPSG:3857"
+					"projection": "EPSG:3857",
+					"constrainResolution": true
```

Comment:
the shogun backend does not set this property, so it does not help much to fake it here

Reply:
wip ;)

Reply:
sry, seen 2 late ^^